packages/admin-panel/src/widgets/PageHeader.jsx

```diff
@@ -20,6 +20,14 @@ const Wrapper = styled.div`
   border-bottom: 1px solid ${({ theme }) => theme.palette.grey['400']};
 `;
 
+const CreateButtonComponent = ({ resourceName, createConfig }) => {
```

Comment:
I was working in this file and ended up refactoring for more readability. I prefer a JSX component over 2 helper functions.